src/backend/utils/misc/guc_gp.c

```diff
@@ -316,6 +316,7 @@ bool		optimizer_enable_direct_dispatch;
 bool		optimizer_enable_hashjoin_redistribute_broadcast_children;
 bool		optimizer_enable_broadcast_nestloop_outer_child;
 bool		optimizer_enable_streaming_material;
+bool        optimizer_enable_offload_entry_to_qe;
```

Comment:
optimizer_XXX typically means GUCs for ORCA

Reply:
should we renamed this to just `enable_offload_entry_to_qe`?

Reply:
OK